src/main/java/net/sf/jabref/model/entry/BibEntry.java

```diff
@@ -194,7 +202,6 @@ public void setType(EntryType type) {
     /**
      * Returns the contents of the given field, or null if it is not set.
      */
-    @Deprecated //Use getFieldOptional instead
```

Comment:
This method is used for example in https://github.com/JabRef/jabref/pull/1451/files#diff-1f5b83f0638c8e3856ca20333099555fR180. The old method ist quite applicable (see https://github.com/JabRef/jabref/pull/1451/files#diff-1f5b83f0638c8e3856ca20333099555fR517), cause `null` object is going to be translated to sql `NULL`. In my opinion the old method is still stable in some cases.